keps/sig-auth/4412-projected-service-account-tokens-for-kubelet-image-credential-providers/README.md

```diff
@@ -1023,13 +1054,10 @@ and operation of this feature.
 Recall that end users cannot usually observe component logs or access metrics.
 -->
 
-- [ ] Events
-  - Event Reason: 
-- [ ] API .status
-  - Condition name: 
-  - Other field: 
-- [ ] Other (treat as last resort)
-  - Details:
+Users can observe events for successful image pulls that use the service account token for image pull.
```

Comment:
Events can be throttled and are sometimes missed. Is there anything in addition one can use here?

I guess if the credential provider is misbehaving the pods won't get pulled?

Reply:
> Events can be throttled and are sometimes missed. Is there anything in addition one can use here?

For registries or images configured to be pulled using a credential provider with a service account, a successful image pull seems to be the only way to confirm that it's working.

> I guess if the credential provider is misbehaving the pods won't get pulled?

This is the case if the image doesn't already exist on the node or if `imagePullPolicy` is `Always`.